common/browser_context.go

```diff
@@ -82,6 +84,7 @@ func (b *BrowserContext) AddInitScript(script goja.Value, arg goja.Value) {
 	hasArgs := arg != nil && !goja.IsUndefined(arg) && !goja.IsNull(arg)
 
 	var source string
+	//nolint:exhaustive
```

Comment:
Keep this in mind while adding nolint: https://github.com/grafana/xk6-browser/issues/58 :)

Reply:
Got it! Does that mean that we should try to apply the lint suggestions? In this case there are quite a few missing cases and for me it was "polluting" the code, especially considering that we have a default case.